### ISSUE TITLE
FileNotFoundError when run distructRerun.py with example data

### ISSUE THREAD

**Issue by xndless** [stevemussmann/admixturePipeline#12]
(base) [XXX admixturePipeline]$ ./distructRerun.py -a example_admixturePipeline_result/ -d example_files/1516030453 -k 1 -K 2
example_files/1516030453/AdmixturePopIdToPopName Exists
example_files/1516030453/K=1/MajorCluster/CLUMPP.files Exists
example_files/1516030453/K=1/MajorCluster/CLUMPP.files/ClumppIndFile.output Exists
example_files/1516030453/K=1/MajorCluster/CLUMPP.files/ClumppPopFile Exists
example_files/1516030453/K=2/MajorCluster/CLUMPP.files Exists
example_files/1516030453/K=2/MajorCluster/CLUMPP.files/ClumppIndFile.output Exists
example_files/1516030453/K=2/MajorCluster/CLUMPP.files/ClumppPopFile Exists
example_files/1516030453/K=2/MinorCluster1/CLUMPP.files/ClumppIndFile.output Exists
example_files/1516030453/K=2/MinorCluster1/CLUMPP.files/ClumppPopFile Exists
Traceback (most recent call last):
  File "./distructRerun.py", line 83, in <module>
    main()
  File "./distructRerun.py", line 61, in main
    c.getMajorClusterCVvalues(input.args.majc)
  File "/media/sdc/XXX/biosoftware/admixturePipeline/clumpp.py", line 154, in getMajorClusterCVvalues
    with open(filepath, 'r') as cvin:
FileNotFoundError: [Errno 2] No such file or directory: 'example_admixturePipeline_result/example.1_8.stdout'

What the matter ?
And can find distruct in ~/miniconda3/bin/

Thanks !!!

**Comment by xndless** [comment]
(base) [XXX admixturePipeline]$ ./distructRerun.py -a example_admixturePipeline_result/ -d example_files/1516030453/ -k 1 -K 2 -c /media/sdc/XXX/biosoftware/distruct1.1/ColorBrewer/
example_files/1516030453/AdmixturePopIdToPopName Exists
example_files/1516030453/K=1/MajorCluster/CLUMPP.files Exists
example_files/1516030453/K=1/MajorCluster/CLUMPP.files/ClumppIndFile.output Exists
example_files/1516030453/K=1/MajorCluster/CLUMPP.files/ClumppPopFile Exists
Traceback (most recent call last):
  File "./distructRerun.py", line 83, in <module>
    main()
  File "./distructRerun.py", line 67, in main
    d.writeDrawparams(drawp, popq, indivq, str(k), outfile, c.pops, c.inds, input.args.width)
  File "/media/sdc/XXX/biosoftware/admixturePipeline/distruct.py", line 62, in writeDrawparams
    fh.write(self.cbdict[self.colorbrew])
KeyError: '/media/sdc/XXX/biosoftware/distruct1.1/ColorBrewer/'

**Comment by xndless** [comment]
(base) [XXX test]$ ../distructRerun.py -a ../example_files/example_admixturePipeline_result -d ../example_files/1516030453 -k 1 -K 10
../example_files/1516030453/AdmixturePopIdToPopName Exists
../example_files/1516030453/K=1/MajorCluster/CLUMPP.files Exists
../example_files/1516030453/K=1/MajorCluster/CLUMPP.files/ClumppIndFile.output Exists
../example_files/1516030453/K=1/MajorCluster/CLUMPP.files/ClumppPopFile Exists
../example_files/1516030453/K=2/MajorCluster/CLUMPP.files Exists
../example_files/1516030453/K=2/MajorCluster/CLUMPP.files/ClumppIndFile.output Exists
../example_files/1516030453/K=2/MajorCluster/CLUMPP.files/ClumppPopFile Exists
../example_files/1516030453/K=2/MinorCluster1/CLUMPP.files/ClumppIndFile.output Exists
../example_files/1516030453/K=2/MinorCluster1/CLUMPP.files/ClumppPopFile Exists
Traceback (most recent call last):
  File "../distructRerun.py", line 83, in <module>
    main()
  File "../distructRerun.py", line 62, in main
    c.getMinorClusterCVvalues()
  File "/media/sdc/XXX/biosoftware/admixturePipeline/clumpp.py", line 177, in getMinorClusterCVvalues
    with open(filepath, 'r') as cvin:
FileNotFoundError: [Errno 2] No such file or directory: '../example_files/example_admixturePipeline_result/spd_bonneville.2_19.stdout'

**Comment by stevemussmann** [comment]
Thanks for bringing this to my attention - some of the updates I made to AdmixPipe in v3 are no longer compatible with the example results files that I uploaded for v2. I'll have to take a look at what is happening and see if I can modify the example result file accordingly. Unfortunately, I'm in the middle of moving across the country and changing jobs right now, so I will not have time to do this for at least a few weeks.
